optuna/trial/_trial.py

```diff
@@ -387,25 +386,58 @@ def objective(trial):
                 Upper endpoint of the range of suggested values. ``high`` is included in the range.
             step:
                 A step of discretization.
+
+                .. note::
+                    Note that :math:`\\mathsf{high}` is modified if the range is not divisible by
+                    :math:`\\mathsf{step}`. Please check the warning messages to find the changed
+                    values.
+
+                .. note::
+                    Specifically, the method returns one of the values in the sequence
+                    :math:`\\mathsf{low}, \\mathsf{low} + \\mathsf{step}, \\mathsf{low} + 2 *
+                    \\mathsf{step}, \\dots, \\mathsf{low} + k * \\mathsf{step} \\le
+                    \\mathsf{high}`, where :math:`k` denotes an integer.
+
+                Raises:
```

Comment:
Seems like this `Raises:` section is too indented. Could you shift it to the same level as `Args:`? This also applies to other occurrences of `Raises:` in this PR.

Reply:
We can probably remove this part now.